packages/node-http-handler/src/set-socket-keep-alive.ts

```diff
@@ -0,0 +1,18 @@
+import { ClientRequest } from "http";
+
+const DEFAULT_INITIAL_DELAY = 10 * 1000;
+
+export interface SocketKeepAliveOptions {
+    keepAlive: boolean;
+    initialDelay: number;
+}
+
+export const setSocketKeepAlive = (request: ClientRequest, socketKeepAlive: SocketKeepAliveOptions) => {
+    if (!socketKeepAlive || socketKeepAlive.keepAlive === false) {
```

Comment:
this should be `if (!(socketKeepAlive?.keepAlive === true))`

strict type checks should be for the opt-in case

Reply:
Done!